app/assets/javascripts/dance_comments/new.js

```diff
@@ -17,35 +17,33 @@ class NewComment{
         <footer>
           <ul>
             <li><a href="/dance_comments/${this.comment[0]}/edit">Edit</a></li>
-            <li id="remove_item"><a href="">Delete</a></li>
+            <li><a href="">Delete</a></li>
           </ul>
         </footer>
       </article>
     </li>`;
   }
 }
 
-let prependTarget = (target, text) => {
-  target.prepend(text);
-};
-
-submitClick = (event) => {
+submitDCClick = (event) => {
   event.preventDefault();
 
-  let tId = $('#dance_comment_comment_type_id').val();
-  let com = $('#dance_comment_comment').val();
-  let uId = $('#user-id').text();
-  let dId = $('#dance-id').text();
+  let comment = {
+    comment: $('#dance_comment_comment').val(),
+    comment_type_id: $('#dance_comment_comment_type_id').val(),
+    user_id: $('#user-id').text(),
+    dance_id: $('#dance-id').text()
+  };
```

Comment:
In future refactoring, I would see if you can encapsulate this functionality into your NewComment class. You create a NewComment object before firing off the AJAX request, and then in the AJAX request you could do something like this:

`data: { dance_comment: newCom.toJSON()`